localstack/services/apigateway/patches.py

```diff
@@ -378,7 +365,7 @@ def method_response_apply_operations(self, patch_operations):
         for param, value in params.items():
             for param_prefix in bool_params_prefixes:
                 if param.startswith(param_prefix) and not isinstance(value, bool):
-                    params[param] = str(value) in ["true", "True"]
+                    params[param] = str(value) in {"true", "True"}
```

Comment:
i suppose this was a spur-of-the-moment change, but i'm curious to learn what the motivation was to change this? do you think this is more readable, or more performant?

Reply:
Yep, I call it boyscout rule :)

To me is natural to do this lookups on sets (hashable) and ends up being more performant, for small sets is negligible, but still natural.